Documentation/JSON schema guide.md

```diff
@@ -41,12 +41,13 @@ All mandatory fields should be present, otherwise the TabBar `Component` won't b
 
 ### Tab meta schema
 
-All mandatory fields should be present, otherwise the Tab `Component` won't be build and thus not added to the TabBar `Component`.
+`title` and `icon_name` are mandatory fields and must be present, otherwise the Tab `Component` won't be build and thus not added to the TabBar `Component`.
 
 | Key | Type | Description | Maps to | Optional | Default value |
 | --- | ---- | ----------- | ------- | -------- | ------------- |
 | `title` | `String` | The title to display on the tab. | `title` | No | . |
 | `icon_name` | `Int` | The name of the icon to display on the tab. | `iconName` | No | . |
+| `show_nav_bar` | `Bool` | `True` if the top navigation bar should be displayed and controlled by the Tab component, `false` otherwise. | `showNavBar` | Yes | `true` |
```

Comment:
Thanks for the feedback! The tab component is not only the text+icon at the bottom - it also hosts it's children. As far as I understood any iOS ViewController can manage the navigation bar, at the moment it's the `navigation` element which actually is the closed source version of the `tab` component. 
Anyway I'll close this issue since it's not related to the `tab` component. @ChristianOrgler @mathiasAichinger Please align on this and make sure the closed source version supports this feature and can be used by other apps.